src/view/popup-film-view.js

```diff
@@ -0,0 +1,194 @@
+import { datePopup, createElement } from '../utils/helpers.js';
+import cn from 'classnames';
+
+const createPopupFilmTemplate = ({
+  title,
+  description,
+  img,
+  genre,
+  country,
+  rating,
+  director,
+  actors,
+  writers,
+  colorRating,
+  isWatchlist,
+  isWatched,
+  isFavorite,
+  releaseDate,
+  duration,
+  comment,
+  commentImg,
+  commentName,
+  originalTitle,
+  ageRating,
+  commentCount,
+}) => {
+
+  const classesWatchlist = cn(
+    'film-card__controls-item',
+    'film-card__controls-item--add-to-watchlist', {
+      'film-card__controls-item--active': isWatchlist
+    });
+
+  const classesWatched = cn(
+    'film-card__controls-item',
+    'film-card__controls-item--mark-as-watched', {
+      'film-card__controls-item--active': isWatched
+    });
+
+  const classesFavorite = cn(
+    'film-card__controls-item',
+    'film-card__controls-item--favorite', {
+      'film-card__controls-item--active': isFavorite
+    });
+
+  const controlsItemButton = (
+    `<button class="film-card__controls-item ${classesWatchlist}"
+      type="button">Add to watchlist</button>
+      <button class="film-card__controls-item ${classesWatched}"
+      type="button">Mark as watched</button>
+      <button class="film-card__controls-item ${classesFavorite}"
+      type="button">Mark as favorite</button>`
```

Comment:
```suggestion
      type="button">
        Mark as favorite
      </button>`
```

И тебе не кажется, что это можно вынести в отдельную функцию, которую можно использовать в нескольких местах?